composer.json

```diff
@@ -10,8 +10,8 @@
     ],
     "license": "MIT",
     "require": {
-        "php": ">=5.5.0",
-        "elasticsearch/elasticsearch": ">=1.0 <6.0",
+        "php": ">=7.0",
+        "elasticsearch/elasticsearch": "5.3.2",
```

Comment:
@kylerberry Since this is used as a library, we shouldn't put a fixed version of ES here so that those who use this library can decide which compatible version to use.

Reply:
Especially since 5.3.2 is included in the range listed in the dependency.

Reply:
I'd be ok with bumping the minimum version, since 1.0 is quite old.